### ISSUE TITLE
Can’t right click on hat in custom block and choose “Add Comment”

### ISSUE THREAD

**Issue by dcbriccetti** [jmoenig/Snap#3381]
To reproduce:

make a block
add a block below the hat (such as “wait”)
right-click on the bottom block and notice “add comment” appears in the menu
right-click on the hat and notice that it does not

It seems inconsistent that “add comment” would not be available for the hat in a custom block.

**Comment by maryfries** [comment]
Thanks for reporting this, Dave! This is another observation from watching teachers learn to use Snap for the first time and noticing what they struggle with. (Dave is a lead BJC Sparks teacher helping us run the teacher professional development this week.)

**Comment by jmoenig** [comment]
Good point, Dave and Mary. Especially since you can now even programmatically set and get block definition comments. I'll leave this open and add it in one of the next patches. Thanks!
